Snabble/UI/de.lproj/SnabbleLocalizable.strings

```diff
@@ -104,6 +104,8 @@
 
 "Snabble.OK" = "OK";
 
+"Snabble.paydirekt.title" = "paydirekt";
```

Comment:
We use phrase.com for translations. 8caa630bdb1e0966973630cba7c96aeaa5ccd0ca
If you find something like that let me know.